api/database/helpers/ApiEnums.php

```diff
@@ -105,6 +105,19 @@ public static function salaryRanges() : array
     const CANDIDATE_STATUS_EXPIRED = 'EXPIRED';
     const CANDIDATE_STATUS_PLACED_CASUAL = 'PLACED_CASUAL';
 
+    public static function candidateStatuses() : array
+    {
+        return [
+            'AVAILABLE',
```

Comment:
Please use the constants above instead of new strings.

Reply:
[aye aye](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3143/commits/1c8c6018a4bbe347c7742a77b3883235ffa39717)